### PR TITLE
Web service integration option

### PR DESCRIPTION
**Pull request by vinaybhattvienna**
New Web Service option integration to save document.

### CODE REVIEW

File: ModelLibrary/Utility/ExternalWebMethod.cs [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+        {
+            string result;
+            
+            try
+            {
+                // Create a byte array of the data we want to send
+                byte[] byteData = Encoding.UTF8.GetBytes(JsonConvert.SerializeObject(ewsData));
+
+                // Create the web request
+                HttpWebRequest request = (HttpWebRequest)WebRequest.Create(url);
```
Comment (Harwinder-VA) [inline ModelLibrary/Utility/ExternalWebMethod.cs]: call close method of httpwebrequest to free up connection and resource by service

File: ModelLibrary/Utility/ExternalWebMethod.cs [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Xml.Linq;
+using VAdvantage.DataBase;
+using VAdvantage.Model;
+
+namespace VAdvantage.Utility
+{
+    class ExternalWebMethod
```
Comment (Harwinder-VA) [inline ModelLibrary/Utility/ExternalWebMethod.cs]: access modifier should be public.

File: ModelLibrary/Utility/ExternalWebMethod.cs (hunk 1 above)
Comment (vinaybhattvienna) [reply ModelLibrary/Utility/ExternalWebMethod.cs]: Sir request to auto closer hoti hai and response ke liye using use kiya hua h.

File: ModelLibrary/Utility/ExternalWebMethod.cs (hunk 2 above)
Comment (vinaybhattvienna) [reply ModelLibrary/Utility/ExternalWebMethod.cs]: Sir ye internal (default access of class) nai chlega ?

File: ModelLibrary/Utility/ExternalWebMethod.cs (hunk 1 above)
Comment (Harwinder-VA) [reply ModelLibrary/Utility/ExternalWebMethod.cs]: ok

**Pull request merged**